server/src/sim.rs

```diff
@@ -30,14 +31,15 @@ use crate::postcard_helpers::{self, SaveEntity};
 
 pub struct Sim {
     cfg: Arc<SimConfig>,
-    rng: SmallRng,
     step: Step,
-    entity_ids: FxHashMap<EntityId, Entity>,
+    pub rng: SmallRng,
+    pub entity_ids: FxHashMap<EntityId, Entity>,
```

Comment:
`rng` and `entity_ids` don't need to be `pub` here. I think the order and visibility of these fields got changed as part of the revert I suggested earlier.